### DISCUSSION TITLE
How can I imporant an exsiting page?

### DISCUSSION THREAD

**Discussion by tomdavidson** [webstudio-is/webstudio-community#146] [Q&A]
I would like to import an existing page and am not finding anything, nor a code view.

**Comment by kof** [comment]
we don't support vanilla html import atm

**Reply by tomdavidson** [reply]
Thanks for the quick response on this!

**Reply by mmiroshnikov** [reply]
Are there any plans for that?
I would gladly move my workflow fwom Webflow, but there are complex projects (html+css wise, I don't mean js/lottie) that I have there.

**Reply by kof** [reply]
webflow copy-paste is launching today, stay tuned https://x.com/getwebstudio
